### PR TITLE
Cover element should work as add-blocker. 

### PR DESCRIPTION
**Pull request by SergiuTudos**
It should cover elements with dynamic content

### CODE REVIEW

**Comment by SergiuTudos** [comment]
can anyone help me understand why it fails ?

**Comment by HelioGuilherme66** [comment]
@SergiuTudos Re-executed the tests and they are green again. Sometimes it happens :(

**Comment by SergiuTudos** [comment]
Great. Waiting for review

**Comment by aaltat** [comment]
Could you raise an issue, where your need is explained. Project uses issues to track what goes in to the next release.

By a quick look,  code looks OK, but I need to take a look from a bigger screen. Could you also write a test where element is not found?

File: atest/acceptance/keywords/elements.robot
Comment (aaltat) [inline atest/acceptance/keywords/elements.robot]: We should also verify that the `newDiv` is visible. Please also add plank line in the end.

**Review by aaltat — CHANGES_REQUESTED** [review]

**Comment by aaltat** [comment]
Also if you want credit for your work add your name to: https://github.com/robotframework/SeleniumLibrary/blob/master/CHANGES.rst

File: atest/acceptance/keywords/elements.robot
Comment (SergiuTudos) [reply atest/acceptance/keywords/elements.robot]: I added name="covered"  for that div

**Review by aaltat — APPROVED** [review]

**Comment by aaltat** [comment]
Thank you for the pull request. The code looks OK from a mobile screen but I need to take a look from a bigger screen before merging the PR.

**Comment by SergiuTudos** [comment]
Hi,
I don't want to be too insistent, but can you please review this and merge this code. I am going to develop a visual testing approach with Robot. Need to cover dynamic content and Get element screenshot for visual diff.

Thank you

**Comment by aaltat** [comment]
Sorry that it's taken so long, but there has been personal blockers and I have not found time to look this. But next time I do something with the project, this is the first thing.

**Comment by aaltat** [comment]
Updated the documentation with few minor fixes and when Travis build is green I will merge the PR to master.

**Pull request merged**
